config/initializers/better_html.rb

```diff
@@ -1,3 +1,5 @@
+return if Rails.env.production?
```

Comment:
Would you mind slipping in a comment above to explain _why_ we skip this in production?

```suggestion
# BetterHtml gem is only present in development and test
return if Rails.env.production?
```